Vagrantfile

```diff
@@ -152,15 +160,16 @@ Vagrant.configure("2") do |config|
             ".git/objects",
             ".git/modules/third-party/objects"
           ]
+      else
+        # Private network for NFS
+        build.vm.network :private_network, ip: "192.168.56.101"
```

Comment:
Ping, if we can get the other NFS related settings moved here too, we can merge this.